Doc/library/traceback.rst

```diff
@@ -29,7 +30,18 @@ which are assigned to the :attr:`~BaseException.__traceback__` field of
    Module :mod:`pdb`
       Interactive source code debugger for Python programs.
 
-The module defines the following functions:
+In the following, we first introduce a number of module-level functions that
+offer basic functionality. They are sufficient for many use cases, including
+interactive inspection of exceptions and tracebacks. We then describe the
+:class:`TracebackException` class and its two helper classes
+:class:`StackSummary` and :class:`FrameSummary`. They offer both more
+flexibility in the output generated and the ability to store the information
+necessary for later formatting without holding references to actual exception
+and traceback objects.
```

Comment:
I think this could be said more concisely and directly. How about something like this?

```suggestion
The module's API can be divided into two parts:
* Module-level functions offering basic functionality such as interactive
  inspection of exceptions and tracebacks.
* :class:`TracebackException` class and its helper classes
  :class:`StackSummary` and :class:`FrameSummary`. These offer both more
  flexibility in the output generated and the ability to store the information
  necessary for later formatting without holding references to actual exception
  and traceback objects.
```